controllers/user/login.js

```diff
@@ -54,7 +55,6 @@ const login = async (req, res, next) => {
   } catch (err) {
     return next(
       createError({
-        message: 'Could not create new user',
```

Comment:
The `message` property should stay, so the error data structure is consistent across the app. The string should be something relevant tho